sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2626,20 +2643,25 @@ public void populateDisplayData(DisplayData.Builder builder) {
     }
 
     public TableReference getOrCreateTable(BigQueryOptions options, String tableSpec)
-        throws IOException {
+        throws InterruptedException, IOException {
```

Comment:
fyi, this change might not be able to be backported to Dataflow if the DoFn is public.